tock/utilization/utils.py

```diff
@@ -33,19 +33,22 @@ def _build_utilization_query(users=None,recent_periods=None, fiscal_year=False):
     """
 
     if not fiscal_year:
-        period_filter =  _limit_to_recent_periods(recent_periods)
+        filter =  _limit_to_recent_periods(recent_periods)
```

Comment:
Opportunity here to tweak just a _bit_ more and append an underscore to differentiate this from the python built-in [filter](https://docs.python.org/3/library/functions.html#filter)